ucore/sanoid.spec

```diff
@@ -0,0 +1,141 @@
+%global version 2.2.0
+%global git_tag v%{version}
+
+# Enable with systemctl "enable sanoid.timer"
+%global _with_systemd 1
+
+Name:		   sanoid
+Version:	   %{version}
```

Comment:
The version global is defined by upstream to be `2.2.0` on line 1 of this file. It's also used in the git_tag and Source URL, so we can't mess that value.

However, on this line, we can append to it. Per my comment in this review on the changelog line, here's where we can append `-ucore1`.

So I think this should look like:
```
Version:	   %{version}-ucore1
```